lis/surfacemodels/land/ac.7.2/ac72_main.F90

```diff
@@ -0,0 +1,1249 @@
+            call SetStressSenescence(AC72_struc(n)%ac72(t)%StressSenescence)
+            call SetDayFraction(AC72_struc(n)%ac72(t)%DayFraction)
+            call SetGDDayFraction(AC72_struc(n)%ac72(t)%GDDayFraction)
+            call SetCGCref(AC72_struc(n)%ac72(t)%CGCref)
+            call SetGDDCGCref(AC72_struc(n)%ac72(t)%GDDCGCref)
+            call SetTimeSenescence(AC72_struc(n)%ac72(t)%TimeSenescence)
+            call SetSumKcTop(AC72_struc(n)%ac72(t)%SumKcTop)
+            call SetSumKcTopStress(AC72_struc(n)%ac72(t)%SumKcTopStress)
+            call SetSumKci(AC72_struc(n)%ac72(t)%SumKci)
+            call SetCCoTotal(AC72_struc(n)%ac72(t)%CCoTotal)
+            call SetCCxTotal(AC72_struc(n)%ac72(t)%CCxTotal)
+            call SetCDCTotal(AC72_struc(n)%ac72(t)%CDCTotal)
+            call SetGDDCDCTotal(AC72_struc(n)%ac72(t)%GDDCDCTotal)
+            call SetCCxCropWeedsNoSFstress(AC72_struc(n)%ac72(t)%CCxCropWeedsNoSFstress)
+            call SetWeedRCi(AC72_struc(n)%ac72(t)%WeedRCi)
+            call SetCCiActualWeedInfested(AC72_struc(n)%ac72(t)%CCiActualWeedInfested)
+            call SetfWeedNoS(AC72_struc(n)%ac72(t)%fWeedNoS)
+            call SetZeval(AC72_struc(n)%ac72(t)%Zeval)
+            call SetBprevSum(AC72_struc(n)%ac72(t)%BprevSum)
+            call SetYprevSum(AC72_struc(n)%ac72(t)%YprevSum)
+            call SetSumGDDcuts(AC72_struc(n)%ac72(t)%SumGDDcuts)
+            call SetHItimesBEF(AC72_struc(n)%ac72(t)%HItimesBEF)
+            call SetScorAT1(AC72_struc(n)%ac72(t)%ScorAT1)
+            call SetScorAT2(AC72_struc(n)%ac72(t)%ScorAT2)
+            call SetHItimesAT1(AC72_struc(n)%ac72(t)%HItimesAT1)
+            call SetHItimesAT2(AC72_struc(n)%ac72(t)%HItimesAT2)
+            call SetHItimesAT(AC72_struc(n)%ac72(t)%HItimesAT)
+            call SetalfaHI(AC72_struc(n)%ac72(t)%alfaHI)
+            call SetalfaHIAdj(AC72_struc(n)%ac72(t)%alfaHIAdj)
+            call SetNextSimFromDayNr(AC72_struc(n)%ac72(t)%NextSimFromDayNr)
+            call SetDayNr1Eval(AC72_struc(n)%ac72(t)%DayNr1Eval)
+            call SetDayNrEval(AC72_struc(n)%ac72(t)%DayNrEval)
+            call SetLineNrEval(int(AC72_struc(n)%ac72(t)%LineNrEval,kind=int32))
+            call SetPreviousSumETo(AC72_struc(n)%ac72(t)%PreviousSumETo)
+            call SetPreviousSumGDD(AC72_struc(n)%ac72(t)%PreviousSumGDD)
+            call SetPreviousBmob(AC72_struc(n)%ac72(t)%PreviousBmob)
+            call SetPreviousBsto(AC72_struc(n)%ac72(t)%PreviousBsto)
+            call SetStageCode(AC72_struc(n)%ac72(t)%StageCode)
+            call SetPreviousDayNr(AC72_struc(n)%ac72(t)%PreviousDayNr)
+            call SetNoYear(AC72_struc(n)%ac72(t)%NoYear)
+            call SetWaterTableInProfile(AC72_struc(n)%ac72(t)%WaterTableInProfile)
+            call SetStartMode(AC72_struc(n)%ac72(t)%StartMode)
+            call SetGDDayi(AC72_struc(n)%ac72(t)%GDDayi)
+            call SetNoMoreCrop(AC72_struc(n)%AC72(t)%NoMoreCrop)
```

Comment:
Awe, what a list...

Reply:
Yes, I was hesitating to organize it alphabetically but that's the way it's organized in my head now :D